T18 feedback

Ali-Alkhars opened the pull request:
Hello, extremely hot reviewers. I hope you still have some energy left!

I am done with this ticket. Take your time to test and play with it.

This ticket has been developed while two API routes needed are being developed, so I made mock routes with mock parameters. Therefore, to start testing you need to do a couple of things:

1. Clone the branch I made in the API repo called "Ali-testing"
2. Host the API by running "node app.js"
3. Clone this branch "T18-Feedback"
4. Host the Bot by running "node index.js"
5. Go to the "complete-a-class" channel on discord and start testing

Have a wonderful day/night depending on when you see this.

![image](https://user-images.githubusercontent.com/34315087/189981334-560349bb-655f-4819-bb6d-e9ee035cf075.png)

Ali-Alkhars committed:
feedback channel and start button

Ali-Alkhars committed:
in progress

Ali-Alkhars committed:
Full mock is available now

Ali-Alkhars committed:
Added a GET request

Ali-Alkhars committed:
About to be done

Ali-Alkhars committed:
Ticket Done

AymenBerb commented:
Amazing work. 

Code looks nice and clean. 

Quick question, where is the command file in the T18-feedback ?

AymenBerb reviewed (approved):
Don't forget to add comments about the initial button.

Ali-Alkhars committed:
Bugs fixed + Info added

- The bot now doesn't crash when the API crashes mid submitting a feedback.

- Changed the wording for the classId label in the form.

- Added the code for the initial feedback message (not called because it should only be called once).

Ali-Alkhars committed:
Added commands folder

absirgue reviewed (approved):
Very very nicely done monsieur. Let's keep in touch when the api endpoints are all created to make sure it still fits
